src/local/index.js

```diff
@@ -323,6 +349,10 @@ class Local implements Side {
 
   // Delete a file or folder from the local filesystem
   destroy (doc, callback) {
+    if (inRemoteTrash(doc)) {
+      callback(null)
+      return
+    }
```

Comment:
It was not easy to fix a case without breaking another.
Which makes me wonder whether we should bring back some integration tests.